### ISSUE TITLE
Mukilteo PD

### ISSUE THREAD

**Issue by hejokelly** [moxboxwa/wa-pursuit-pdr-data#90]
[x] PDR sent (6/7/24 - hk)
[x] Receipt of PDR confirmed by autoresponse: https://arlingtonwa.mycusthelp.com/WEBAPP/_rs/(S(kbgwyl0eyojfqztk111ad5mr))/SupportHome.aspx
[x] Response received
[ ] ETA received (7/5/24)
[x] Agency requests clarification (6/14/24)
[ ] Data received (installment #/final)
[ ] Data uploaded
[ ] Harmonized file created
[ ] Report produced

**Comment by hejokelly** [comment]
@moxboxwa we got the data in two formats - an Excel spreadsheet that came to me by email attachment and an Excel spreadsheet uploaded to their PDR portal. She sent the attachment thinking that the data coming thru the portal would be a PDF, and therefore of limited utility. But they're both Excel files. I have not confirmed that they contain the same data.

**Comment by moxboxwa** [comment]
We need to f/u to get clarification on the "Dispositions" field codes
![image](https://github.com/user-attachments/assets/dcd8b937-03cc-4047-b5f2-035d6e1ade0d)

**Comment by moxboxwa** [comment]
Also, from the correspondence they say:
![image](https://github.com/user-attachments/assets/e2305adc-a1f0-4736-b222-ab483a224fa0)

We should request a copy of this spreadsheet.

**Comment by hejokelly** [comment]
@moxboxwa I have requested clarification about whether the spreadsheet previously provided is the one he references in his email and, if not, asked them to provide it.

**Comment by hejokelly** [comment]
@moxboxwa they replied with two spreadsheets. Let me know if they seem to correspond to his email.

**Comment by moxboxwa** [comment]
2 data files sent:

* engaged pursuits 2022-current -- internal tracking file with pursuits the agency was actively involved in, started in 2022 (n=8; 3 from 2022-23, 5 from 2024)
* dispatched pursuits 2019-2023 -- dispatch records of pursuits they were called to.  They may not have engaged (n=31; 21 from 2022-2023, others are earlier).  If they did engage, it looks like there is a case number recorded.  Only 5 incidents have case numbers.

Of the 3 engaged pursuits from 2022-23, only one shows up on the dispatch record.  The remaining 20 dispatch pursuits are not included in the engaged file.  So, effectively no overlap for this period.  In the earlier period there are 4 case numbers in the dispatch file, so we can assume these are the incidents for this time period.

Bottom line:  we can start with these datafiles.  Still need clarification on the dispatch file "Disposition" codes (see above).
